### ISSUE TITLE
sXe Injecting Bypass

### ISSUE THREAD

**Issue by hasanalizxc** [rootm0s/Injectors#3]
Dear rootm0s,

Hope you are fine. I looked the list and very nice, thanks. But i have a problem about sXe injected software. Maybe you heard. sXe blocks the DLL injecting to the hl.exe (this is example, you can run several games with sXe) and cheat doesn't work. Is there any solution for this? If you know, please share.

Thanks.

**Comment by rootm0s** [comment]
What game are you trying to inject a DLL to? I suppose you can design a driver (ring0) that handles all the injecting/reading memory. Not that into coding cheats or anti-cheats nowdays.

Read more here:
https://github.com/rootm0s/Injectors/tree/master/ImprovedReflectiveDLLInjection
https://github.com/rootm0s/Cheats/tree/master/ring0

**Comment by hasanalizxc** [comment]
Thank you so much your interest and help. Firstable i don't have requirement about programming and can't design. I installed sXe injected 17.2 and opened it, minimized. Then opened CS 1.6 (version 4554), when i opened CS, sXe injecting itself to the hl.exe.. Then i am trying to inject NorAdrenaline DLL with Extreme Injector 3.73 but i encountered "null returned exception" error and couldn't inject DLL and i saw in CS console "INJECT COMMAND OFFLINE".

I also found a topic in a website:

https://www.unknowncheats.me/forum/counterstrike-1-5-1-6-and-mods/286552-anya-disabler.html

But in this time i entered PatchClient command and it works. But when i enter PatchOpengl and other commands, game crashes. I tried several times but result is same.

If you help me, i appreciate you..

Thanks.
